package.json

```diff
@@ -12,6 +12,7 @@
 	"scripts": {
 		"lint": "xo",
 		"test": "npm run lint && flow check test/flow-types && tsc --noEmit -p test/ts-types && nyc tap --no-cov --timeout=300 test/*.js test/reporters/*.js test/integration/*.js",
+        "t": "npm run lint && flow check test/flow-types && tsc --noEmit -p test/ts-types && nyc tap --no-cov --timeout=300 test/load-config.js",
```

Comment:
You can type `npm t` rather than `npm run test` if you prefer. (`npm test` works as well.)

Reply:
Absolutely I'll remove that. Right now it's just a command so that I don't have to parse all test output. That being said I'll run all tests before I push an update.